common/src/model_deployment/deploy.py

```diff
@@ -149,19 +157,12 @@ def parse_args():
         parser.error("Arg max_concurrent_requests_per_instance cannot be less than 1")
     if args.request_timeout_ms < 1 or args.request_timeout_ms > 90000:
         parser.error("Arg request_timeout_ms should lie between 1 and 90000")
-    if args.max_queue_wait_ms < 1 or args.max_queue_wait_ms > 500:
-        parser.error("Arg max_queue_wait_ms should lie between 1 and 500")
+    if args.max_queue_wait_ms < 1 or args.max_queue_wait_ms > 90000:
+        parser.error("Arg max_queue_wait_ms should lie between 1 and 90000")
```

Comment:
Is 60000 or 90000 the max? Consider creating a constant to ensure these values are consistent.

Reply:
Its 90K. Updated at places where 60K was mentioned as max value.

Reply:
Could we dynamically read this variable from somewhere? 

Also +1 to @elesel's suggestion on using a constant.

Reply:
> Could we dynamically read this variable from somewhere?

Not sure, let me figure out and add it in another PR. Whatever public docs we have just show default values and not max value that can be accepted

@skasturi